plugins/nvme/nvme.quirk

```diff
@@ -54,6 +54,7 @@ Flags = needs-shutdown
 # Lexar
 [NVME\VEN_1D97]
 Flags = ensure-semver,commit-ca3
+NvmeCommitSlot = 1
```

Comment:
is this always going to be true?  It seems like if there is a slot 0 and a slot 1 you would want to flash _the one you're not booted into._  Can you detect the slot in use?

Reply:
Copy/paste from Lexar:

It depends on the slot number which SSD Firmware is supported.  You can use nvme id-ctrl command to check the available slot number. For example below:
 
The parameter frmw shows the supported slot number, 0x1a = 00011010, the value of bit3~1 is the supported slot number. So value 101 equals to 5 means the supported slot number of this NVME SSD is 5, then you can use slot 1~5 either for the parameter. Why we used slot 1 is that this slot number is always existed and available. J

<img width="235" height="479" alt="image" src="https://github.com/user-attachments/assets/aefc3057-409b-4ac8-a8d4-5c857bbbd513" />

Reply:
Okay, alternate proposal in https://github.com/fwupd/fwupd/pull/9501 -- which I like a lot more than this one!